code/modules/events/ghost_role/fugitive_event.dm

```diff
@@ -98,9 +104,11 @@
 	new /obj/item/autosurgeon(landing_turf)
 
 //security team gets called in after 10 minutes of prep to find the refugees
-/datum/round_event/ghost_role/fugitives/proc/spawn_hunters()
-	var/backstory = pick(HUNTER_PACK_COPS, HUNTER_PACK_RUSSIAN, HUNTER_PACK_BOUNTY, HUNTER_PACK_PSYKER)
-	var/datum/map_template/shuttle/ship
+/datum/round_event/ghost_role/fugitives/proc/spawn_hunters(backstory)
+	var/list/candidates = poll_ghost_candidates("Do you wish to be considered for a fugitive hunting crew?", ROLE_FUGITIVE_HUNTER)
```

Comment:
would it be worthwhile to include the type here?

Reply:
I would say because it would mean I need to add another switch, but it would also probably be nice to warn players before throwing them into the game as a Psyker.

Reply:
is there any reason these defines can't be like, human readable names?

Reply:
Not that I'm aware of!